src/main/java/game/applicationBase/LocalFileTool.java

```diff
@@ -0,0 +1,137 @@
+package game.applicationBase;
+
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+import game.Game;
+import game.stores.ActionStore;
+import game.stores.ItemStore;
+import game.stores.SceneStore;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+
+public class LocalFileTool {
+    // TODO: 06-03-2020 clean this mess
+    private static final String SLASH = System.getProperty("file.separator");
+    private static final String MAIN_DIR = System.getProperty("user.home") + SLASH + ".spork" + SLASH ;
+    private static final String GAME_DIR = MAIN_DIR + "data" + SLASH;
+    private static final String SAVE_DIR = MAIN_DIR + "saves" + SLASH;
+
+    private static File mainItemFile = new File(GAME_DIR + "main-game.items.json");
+    private static File mainSceneFile = new File(GAME_DIR + "main-game.scenes.json");
+    private static File mainActionsFile = new File(GAME_DIR + "main-game.actions.json");
+
+    public static Game fromFile(String fileName) throws FileNotFoundException {
+        fileName = MAIN_DIR + fileName;
+        Gson gson = new Gson();
+        JsonReader jsonReader = new JsonReader(new FileReader(fileName));
+
+        return gson.fromJson(jsonReader, Game.class);
+    }
+
+    public static List<File> listSaveFiles(ClassLoader classLoader) {
+        setupDirectories(classLoader);
+        File saveDir = new File(SAVE_DIR);
+        return Arrays.asList(saveDir.listFiles());
+    }
+
+    public static Game makeNewGameFromFile(ClassLoader classLoader) throws FileNotFoundException {
+        return makeNewGameFromFile(classLoader, new File(GAME_DIR + "main-game.json"));
+    }
+
+    public static Game makeNewGameFromFile(ClassLoader classLoader, File file) throws FileNotFoundException {
```

Comment:
The function is overloaded like this because we plan on allowing different game files to be loaded. People can then make their own json files for the game :)